tests/Makefile.sub

```diff
@@ -1,58 +1,8 @@
-TESTS = \
-
-include $(JLM_ROOT)/tests/jlm/Makefile.sub
-
-TEST_SOURCES = \
+libjlmtest_SOURCES = \
 	tests/test-operation.cpp \
 	tests/test-registry.cpp \
 	tests/test-runner.cpp \
 	tests/test-types.cpp \
 	tests/TestRvsdgs.cpp \
-	$(patsubst %, tests/%.cpp, $(TESTS))
-
-$(JLM_BUILD)/tests/test-runner: rvsdg-debug
-$(JLM_BUILD)/tests/test-runner: CXXFLAGS += $(CXXFLAGS_DEBUG)
-$(JLM_BUILD)/tests/test-runner: CPPFLAGS += -I$(JLM_ROOT)/tests -I$(JLM_ROOT) -I$(shell $(LLVMCONFIG) --includedir)
-$(JLM_BUILD)/tests/test-runner: LDFLAGS=-L$(JLM_BUILD) $(shell $(LLVMCONFIG) --ldflags --libs --system-libs) -ltooling -lhls -lllvm -lrvsdg -lutil
-$(JLM_BUILD)/tests/test-runner: %: $(patsubst %.cpp, $(JLM_BUILD)/%.la, $(TEST_SOURCES)) $(JLM_BUILD)/libtooling.a $(JLM_BUILD)/librvsdg.a $(JLM_BUILD)/libhls.a $(JLM_BUILD)/libllvm.a $(JLM_BUILD)/libutil.a
-	mkdir -p ${dir $@}
-	$(CXX) -o $@ $(filter %.la, $^) $(LDFLAGS)
-
-$(patsubst %, %(JLM_BUILD)/tests/%.la, $(TESTS)): CPPFLAGS += -Itests -I$(shell $(LLVMCONFIG) --includedir)
-
-TESTLOG = true
-
-jlm-check: jlm-check-utests jlm-check-ctests
-
-jlm-check-ctests: jlc-debug jlm-opt-debug
```

Comment:
The C tests seem to have gotten lost completely.

Reply:
yes indeed, they are restored now